SPEC.md

```diff
@@ -378,10 +378,10 @@ Sub-units (e.g. decimals, "0.3", or milli-units, "300m") are NOT permissible.
 
 ## App Container Image Discovery
 
-An app name has a URL-like structure, for example `example.com/reduce-worker`.
-However, there is no scheme on this app name, so it cannot be directly resolved to an App Container Image URL.
-Furthermore, attributes other than the name may be required to unambiguously identify an app (version, OS and architecture).
-App Container Image Discovery prescribes a discovery process to retrieve an image based on the app name and these attributes.
+An app container image name has a URL-like structure, for example `example.com/reduce-worker`.
+However, there is no scheme on this image name, so it cannot be directly resolved to an App Container Image URL.
+Furthermore, attributes other than the name may be required to unambiguously identify an image (version, OS and architecture).
+App Container Image Discovery prescribes a discovery process to retrieve an image based on the image name and these attributes.
```

Comment:
Can you please add a line: "These attributes are expressed in the *labels* field of the ImageManifest"